datumaro/plugins/widerface_format.py

```diff
@@ -44,8 +45,13 @@ def __init__(self, path, subset=None):
 
     def _load_categories(self):
         label_cat = LabelCategories()
-        path = osp.join(self._dataset_dir, WiderFacePath.LABELS_FILE)
-        if osp.isfile(path):
+        if is_meta_file(self._dataset_dir):
```

Comment:
This line looks pretty strange, IMO. The directory path is passed into the function called "is_meta_file".

Reply:
Fixed. I added `is_meta_file_in_dir` function.